### ISSUE TITLE
(2) Train the acoustic model in Journalism dataset [2]

### ISSUE THREAD

**Issue by PanosAntoniadis** [eellak/gsoc2019-sphinx#6]
- [X] Split data into train and test set (70% - 30%)
- [X] Remove punctuation and convert all words to lowercase.
- [x] Extend the dictionary using the training set and Phonetisaurus.
- [x] Train the acoustic model on the train set.
- [x] Evaluate it on the test set.

**Comment by PanosAntoniadis** [comment]
Accuracy in the train set increased only by 2% (67.91% from 66.03%). In order to confirm that the adaptation procedure was correct, I evaluate it on the train set, giving 84.13% accuracy. 

All metrics are uploaded here: https://www.dropbox.com/s/0mpn06j87dsvw6n/Metrics.xlsx?dl=0
